.changeset/mighty-chairs-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-banner": major
```

Comment:
forgot to mention, this would be a `patch` change.
```suggestion
"@khanacademy/wonder-blocks-banner": patch
```

Reply:
@MikeKlemarewski I missed the `major` part in changesets, sorry. You'll need to change that to become `patch`. This PR will need to be reverted.